src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -105,6 +105,20 @@
               + "https://github.com/bazelbuild/remote-apis/blob/master/build/bazel/remote/asset/v1/remote_asset.proto")
   public String remoteDownloader;
 
+  @Option(
+    name = "experimental_remote_downloader_config",
+    defaultValue = "null",
+    documentationCategory = OptionDocumentationCategory.REMOTE,
+    effectTags = {OptionEffectTag.UNKNOWN},
+    help =
+        "Specify a file to configure the remote downloader with. This file consists of lines, "
+            + "each of which starts with a directive (`allow`, `block` or `rewrite`) followed "
+            + "by either a host name (for `allow` and `block`) or two patterns, one to match "
+            + "against, and one to use as a substitute URL, with back-references starting from "
+            + "`$1`. It is possible for multiple `rewrite` directives for the same URL to be "
+            + "give, and in this case multiple URLs will be returned.")
+  public String remoteDownloaderConfig;
```

Comment:
`RemoteOptions` is used for remote cache/execution. Since your changes are focusing on repository downloading, maybe move the flag to `RepositoryOptions`?

Reply:
I grouped this with the other `remote_*` options, including `remote_downloader_header` and `remote_cache_header`. Should they all be moved?

Reply:
`remote_download_header` is used for `GrpcRemoteDownloader` and `remote_cache_header` is used for `GrpcCacheClient`. I think we should keep them there since they are both part of Remote Execution API.

Reply:
I've moved the field.